aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -158,12 +164,107 @@ define(["dojo/_base/declare",
         "alfresco/dialogs/AlfDialog",
         "alfresco/forms/Form",
         "dojo/_base/array",
-        "jquery"],
-        function(declare, AlfCore, lang, AlfDialog, AlfForm, array, $) {
+        "jquery",
+        "dojo/aspect"],
+        function(declare, AlfCore, lang, AlfDialog, AlfForm, array, $, aspect) {
 
    return declare([AlfCore], {
 
       /**
+       * An array of the i18n files to use with this widget.
+       *
+       * @instance
+       * @type {object[]}
+       * @default [{i18nFile: "./i18n/DialogService.properties"}]
+       */
+      i18nRequirements: [{i18nFile: "./i18n/DialogService.properties"}],
+
+      /**
+       * The topic published when the confirmation button is used.
+       * 
+       * @instance
+       * @type {string}
+       * @default
+       */
+      _formConfirmationTopic: "ALF_CREATE_FORM_DIALOG_MIXIN_CONFIRMATION_TOPIC",
+
+      /**
+       * The topic published when the button confirming the current dialog, but requesting to
+       * repeat the dialog is used.
+       * 
+       * @instance
+       * @type {string}
+       * @default 
+       */
+      _formConfirmationRepeatTopic: "ALF_CREATE_REPEATING_FORM_DIALOG_MIXIN_CONFIRMATION_TOPIC",
+
+      /**
+       * The topic published when the cancellation button is used.
+       * 
+       * @instance
+       * @type {string}
+       * @default 
+       */
+      _formCancellationTopic: "ALF_CLOSE_DIALOG",
+
+      /**
+       * The default configuration for form dialogs. This is used as a base when requests are received.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       */
+      defaultFormDialogConfig: {
```

Comment:
This isn't a string, and the @default without a value won't work with complex objects